src/sdgio-edit.owl

```diff
@@ -1601,13 +1600,14 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020237> "sub-outcome of an intermediate development outcome"@en)
 SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00000251>)
 
-# Class: <http://purl.unep.org/sdg/SDGIO_00020300> (cross-cutting objective)
+# Class: <http://purl.unep.org/sdg/SDGIO_00020300> (CGIAR cross-cutting objective)
 
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020300> "A CGIAR objective specification which directs the organisation's mission such that 1) it contributes to the United Nation’s Sustainable Development Goals and 2) each research group contributes to its achievement.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020300> <http://orcid.org/0000-0062-3253-4387>)
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020300> "2019-05-08T20:44:23Z"^^xsd:dateTime)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.unep.org/sdg/SDGIO_00020300> "cross-cutting issue")
-AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020300> "cross-cutting objective"@en)
+AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020300> "")
```

Comment:
There should be no blank comments

Reply:
I took away the blank comment and fixed the UN typo. For the meaning of CC. I will add this problem to issue #164 and fix it as soon as I get more information.